src/System.Runtime.WindowsRuntime/src/Resources/Strings.resx

```diff
@@ -348,4 +348,7 @@
   <data name="UnauthorizedAccess_InternalBuffer" xml:space="preserve">
     <value>MemoryStream's internal buffer cannot be accessed.</value>
   </data>
+  <data name="Argument_NeedNonNegNum" xml:space="preserve">
+    <value>Non-negative number required.</value>
```

Comment:
Usually we have error messages as complete sentences.

So, I would suggest: "A non-negative number is required".